mirgecom/multiphysics/thermally_coupled_fluid_wall.py

```diff
@@ -1575,7 +1562,8 @@ def basic_coupled_ns_heat_operator(
     # Compute the subdomain NS/diffusion operators using the augmented boundaries
 
     my_ns_operator = partial(ns_operator,
-        viscous_numerical_flux_func=viscous_facial_flux_harmonic)
+        viscous_numerical_flux_func=viscous_facial_flux_harmonic,
+        use_esdg=use_esdg)
```

Comment:
@majosm: Is there a problem letting the caller define the `ns_operator` instead of having the ESDG passthrough argument and using the partial?   This construction precludes the caller setting these things for themselves.

Reply:
Not really, I just wanted to keep the implementation as dumb as possible since that function is mostly intended to be used only for tests/examples.